relayer/chains/icon/route.go

```diff
@@ -56,24 +56,24 @@ func (icp *IconProvider) SendTransaction(
 		return nil, err
 	}
 
-	txParamEst := &types.TransactionParamForEstimate{
-		Version:     types.NewHexInt(JsonrpcApiVersion),
-		FromAddress: types.Address(wallet.Address().String()),
-		ToAddress:   types.Address(icp.PCfg.ContractAddress),
-		NetworkID:   types.NewHexInt(int64(icp.PCfg.NetworkID)),
-		DataType:    "call",
-		Data: types.CallData{
-			Method: msg.Method,
-			Params: msg.Params,
-		},
-	}
-
-	step, err := icp.client.EstimateStep(txParamEst)
-	if err != nil {
-		return nil, fmt.Errorf("failed estimating step: %w", err)
-	}
-
-	stepVal, err := step.Int()
+	//txParamEst := &types.TransactionParamForEstimate{
```

Comment:
why was this removed? we'll need this estimate feature on the production.

or

we can have "estimate": true/false on the icon config. If true, use the value from estimateStep, else use custom value.

Reply:
sorry, this was unintentional.